dnf-docker-test/features/steps/rpm_steps.py

```diff
@@ -114,6 +115,11 @@ def pkgs_split(pkgs):
                     rpmdb.remove(pkg_post)
                 state = rpm_utils.analyze_state(pkg_pre, pkg_post)
                 if state != expected_state:
+                    if state == State.unchanged and expected_state == State.reinstalled:
+                        # workaround for unchanged rpmdb timestamp
```

Comment:
this is obviously very bad workaround

Reply:
Neither I don't like searching for a pattern in stdout, do you have any better idea?